crates/datasources/Cargo.toml

```diff
@@ -25,6 +25,7 @@ futures = { workspace = true }
 gcp-bigquery-client = "0.18.0"
 klickhouse = { version = "0.11.2", features = ["tls"] }
 protogen = { path = "../protogen" }
+parser = {path = "../parser"}
```

Comment:
super-nit: we added spaces in other places for this form. 

idle musing: is there a toml linter we could get?